shell-completion/bash/systemctl.in

```diff
@@ -68,35 +68,35 @@ __filter_units_by_properties () {
 
 __get_all_units      () { { __systemctl $1 list-unit-files; __systemctl $1 list-units --all; } \
```

Comment:
Why do not you add `"$2*"` here?

Reply:
There's two reasons why I did not add it:
 - the time difference is negligible
 - the zsh completion does not have it

I'm fine with adding it, although unless you feel strongly I'd do that for zsh and bash on top.